hoomd/md/methods/methods.py

```diff
@@ -96,79 +195,56 @@ class NVT(Method):
             :math:`\eta_\mathrm{rot}`)
     """
 
-    def __init__(self, filter, kT, tau):
+    def __init__(self, filter, thermostat=None):
 
         # store metadata
         param_dict = ParameterDict(filter=ParticleFilter,
                                    kT=Variant,
-                                   tau=float(tau),
+                                   thermostat=Thermostat,
                                    translational_thermostat_dof=(float, float),
                                    rotational_thermostat_dof=(float, float))
+        if thermostat is None:
+            thermostat = ConstantEnergy()
         param_dict.update(
-            dict(kT=kT,
-                 filter=filter,
+            dict(filter=filter,
                  translational_thermostat_dof=(0, 0),
-                 rotational_thermostat_dof=(0, 0)))
+                 rotational_thermostat_dof=(0, 0),
+                 thermostat=thermostat))
```

Comment:
Remove `translational_thermostat_dof` and `rotational_thermostat_dof` from `param_dict` and this call to `update`. These have moved to the `MTTK` thermostat class.